yarn.lock

```diff
@@ -7839,7 +8816,15 @@ watchify@^3.9.0:
     through2 "^2.0.0"
     xtend "^4.0.0"
 
-waypoints@^4.0.0:
+watchpack@^1.4.0:
+  version "1.4.0"
+  resolved "https://registry.yarnpkg.com/watchpack/-/watchpack-1.4.0.tgz#4a1472bcbb952bd0a9bb4036801f954dfb39faac"
+  dependencies:
+    async "^2.1.2"
+    chokidar "^1.7.0"
+    graceful-fs "^4.1.2"
+
+waypoints@^4.0.1:
```

Comment:
I wouldn't imagine a minor version bump like this would be the culprit to the waypoints issue, but seems like there was some breaking API change to the library in this PR.